tests/test_datafiles.py

```diff
@@ -0,0 +1,30 @@
+"""Test installation of PINT data files"""
+import os
+import unittest
+import pint
+
+
+class TestDatafiles(unittest.TestCase):
```

Comment:
This would be nicer in a `pytest` style - just write a function that asserts whatever it is. Or if you want to share the file list, make that a fixture:
```python
@pytest.fixture
def files():
     ... stuff ...
     return list_of_filenames
```
Or maybe:
```python
files = """stuff"""
list_of_files = [f.strip() for f in files.split("\n")]

@pytest.mark.parametrize("filename", list_of_files)
def test_can_read(filename):
    open(filename).read()
```
to get separate tests if you think that's useful.

Reply:
OK, sure.  Easier to read.